ReleaseProcessAutomation/Scripting/MSBuildCallAndCommit.cs

```diff
@@ -56,8 +57,15 @@ public int CallMSBuildStepsAndCommit (MSBuildMode msBuildMode, SemanticVersion v
 
     if (string.IsNullOrEmpty(msBuildPath))
     {
-      _log.Warning("No MSBuild Path specified in config, will continue without MSBuild");
-      _console.WriteLine("There was no MSBuildPath specified in the config, will continue without Invoking MSBuild");
+      _log.Warning("No MSBuild Path specified in config, will continue without MSBuild.");
+      _console.WriteLine("There was no MSBuildPath specified in the config.\nWill continue without invoking MSBuild.");
+      return -1;
+    }
+
+    if (!File.Exists(msBuildPath))
+    {
+      _log.Warning("The configured MSBuildpath '{1}' does not exist.Current directory: '{2}'", msBuildPath, Environment.CurrentDirectory);
+      _console.WriteLine($"The configured MSBuildPath '{msBuildPath}' does not exist.\nCurrent directory: '{Environment.CurrentDirectory}'\nPlease configure a proper MSBuildPath in the config.\nWill continue without invoking MSBuild.");
```

Comment:
Ah, okay. My intention was to use Path.GetFullPath (msBuildPath) which resolves the path. Sorry that this wasn't explicit before. 

Upon further consideration: you could actually do this right at the start, when you read the msBuildPath from the config file.  This removes any effects from changing directories etc during the execution / when spawning new processes.

There's also a missing whitespace in line 67 before 'Current Directory'